web/app_dev.php

```diff
@@ -18,7 +18,10 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+/**
+ * @var Composer\Autoload\ClassLoader $loader
+ */
+$loader = require __DIR__.'/../app/autoload.php';
```

Comment:
The difference is including `bootstrap.php.cache` in `app.php` and `autoload.php` in `app_dev.php`. You're right that both ultimately include `app/autoload.php`, but the risk (if it's even real) is that your `bootstrap.php.cache` is somehow out of sync with your source files, so things work in `app_dev.php` (with the true source files), but don't in `app.php` (which uses the compiled versions). Practically speaking, vendor files are only updated with Composer, which includes the script to re-update `bootstrap.php.cache`, so I'm not sure it's a real issue.